source/_patterns/02-molecules/vue-widget/__tests__/integration/vue-cards.spec.js

```diff
@@ -0,0 +1,85 @@
+import { render, Simulate } from 'vue-testing-library';
```

Comment:
Is `integration/` necessary? Can it be removed to drop a little complexity?

Reply:
yeah, sure thing